features/legacy/reviews/reviews.feature

```diff
@@ -10,16 +10,19 @@ Feature: Reviews
             | beth@foo.com   |
             | martha@foo.com |
             | rick@foo.com   |
+        And there are following customers:
+            | email           |
+            | jon@example.com |
         And the following products exist:
             | name            | price | average rating |
             | Black T-Shirt   | 18.99 | 4              |
             | Sylius Tee      | 12.99 | 5              |
             | Symfony T-Shirt | 15.00 | 1              |
         And there are following reviews:
-            | title      | rating | comment               | author         | product         | subject type | status |
-            | Really bad | 1      | Lorem ipsum dolor sit | beth@foo.com   | Symfony T-Shirt | product      |        |
-            | Very good  | 4      | Lorem ipsum dolor sit | martha@foo.com | Black T-Shirt   | product      |        |
```

Comment:
Watch out with this "Martha" removal ;)
![zrzut ekranu 2016-06-15 o 09 24 53](https://cloud.githubusercontent.com/assets/6212718/16071344/18439f0a-32db-11e6-8aae-56b973673478.png)